jgrapht-core/src/main/java/org/jgrapht/alg/flow/MaximumFlowAlgorithmBase.java

```diff
@@ -60,58 +61,85 @@
      * Default tolerance.
      */
     public static final double DEFAULT_EPSILON = 1e-9;
+    public final double EPSILON; // tolerance (DEFAULT_EPSILON or user-defined)
 
-    private Extension<V, ? extends VertexExtensionBase> vXs;
-    private Extension<E, ? extends EdgeExtensionBase> eXs;
+    protected Graph<V, E> network; //input network
+    protected final boolean DIRECTED_GRAPH; //indicates whether the input graph is directed or not
 
-    abstract DirectedGraph<V, E> getNetwork();
 
-    <VE extends VertexExtensionBase, EE extends EdgeExtensionBase> void init(
+    protected Extension<V, ? extends VertexExtensionBase> vertexExtensionManager;
+    protected Extension<E, AnnotatedFlowEdge> edgeExtensionManager;
+
+    protected double maxFlowValue=-1; //Max flow established after last invocation of the algorithm.
+    protected Map<E, Double> maxFlow=null; //Mapping of the flow on each edge.
```

Comment:
Not sure what makes you unhappy here. Perhaps you prefer /* comment */ better. I'll add that.